### DISCUSSION TITLE
Logging in test not working

### DISCUSSION THREAD

**Discussion by jaimeatstariongroup** [serilog-contrib/serilog-sinks-grafana-loki#168] [Q&A]
Hi, I'm trying to create a test and log into Grafana/Loki using the Microsoft.Extensions.Logging.ILogger but I can't make it work. 

This is the configuration that I'm using:

![image](https://user-images.githubusercontent.com/112856984/225595560-59599363-9765-451b-b88f-085c2e0b7d8f.png)

The path is read from the appsettings.json and it's value is: "http://localhost:3100". And the TestClass is:

![image](https://user-images.githubusercontent.com/112856984/225596096-e6b95688-2f71-434f-9ecb-e0518fa23a6a.png)

What am I missing or doing wrong?

**Comment by mishamyte** [comment]
Hey @jaimeatrhea,

Have you tried to enable `SelfLog` and check it for useful info? ([Section common problem could be useful](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Common-problems#problems)

**Reply by jaimeatstariongroup** [reply]
I've tried enabling the SelfLog but didn't work. However I found the answer here: https://stackoverflow.com/questions/39604415/serilog-selflog-not-outputting-anything 
The problem was basically that I wasn't disposing the Logger. So I added:

![image](https://user-images.githubusercontent.com/112856984/225623250-c1c1dd83-6a3b-40d4-a687-91c647e90feb.png)

Thanks for your help!
